WebCryptoAPI/generateKey/failures.js

```diff
@@ -98,7 +98,7 @@ function run_test(algorithmNames) {
                 results.push({name: algorithmName, hash: "SHA-256", modulusLength: 1024, publicExponent: publicExponent});
             });
         } else if (algorithmName.toUpperCase().substring(0, 2) === "EC") {
-            ["P-512", "Curve25519"].forEach(function(curveName) {
+            ["P-512"].forEach(function(curveName) {
```

Comment:
This still remains a valid failure expectation. Curve25519 is not a valid identifier, there's no need to remove this.

Reply:
There is  patch for that Curve that is not landed yet.